winpr/libwinpr/utils/collections/MessageQueue.c

```diff
@@ -99,7 +101,6 @@ BOOL MessageQueue_Dispatch(wMessageQueue* queue, wMessage* message)
 	queue->tail = (queue->tail + 1) % queue->capacity;
 	queue->size++;
 
-	message = &(queue->array[queue->tail]);
```

Comment:
If this is removed the time is set in the message parameter but not in the message added to the queue.